### ISSUE TITLE
Allow user to manually set the cards

### ISSUE THREAD

**Issue by ExcaliburZero** [mratzloff/tarokka#4]
One usecase that I had for this tool was to be able to present a pre-chosen set of the 5 tarokka cards that I had picked based on where I wanted the 3 items, ally, and Strahd to be found.

Currently it is possible to satisfy this usecase through removing all but the 5 wanteed cards from the deck and reshuffling and locking in cards until all of the cards I want have been assigned to each of the correct roles. However, this approach is somewhat tedious.

It would be nice if there was a way on the DM screen to manually choose/change the cards assigned to each role.

**Comment by Painisio** [comment]
> One usecase that I had for this tool was to be able to present a pre-chosen set of the 5 tarokka cards that I had picked based on where I wanted the 3 items, ally, and Strahd to be found.
> 
> Currently it is possible to satisfy this usecase through removing all but the 5 wanteed cards from the deck and reshuffling and locking in cards until all of the cards I want have been assigned to each of the correct roles. However, this approach is somewhat tedious.
> 
> It would be nice if there was a way on the DM screen to manually choose/change the cards assigned to each role.

If you want a pre set cards for the adventure I recomend to by the cards, 'll be easier to set as you wish! The ideia behind this program is to be randon, like an cartomancer. Its important to know that your comentary its god, but you dont get the idea.

**Comment by ExcaliburZero** [comment]
> If you want a pre set cards for the adventure I recomend to by the cards, 'll be easier to set as you wish! The ideia behind this program is to be randon, like an cartomancer. Its important to know that your comentary its god, but you dont get the idea.

Ah, yep that makes sense that this program is intended for the randomization, that's a fair design choice.

I was looking more at the utility of using it for remote DnD sessions over Roll20 + a Discord call, as a flashier way of doing the card reveals.

I used it for one DnD campaign I ran a while back and my players really enjoyed it. They thought the tool was really cool and made for a really cool reveal of the card readings.

Thanks for making this tool! I really appreciate it.
